m4_sessions/1810-inning/problems_tech_cannot_solve.md

```diff
@@ -18,14 +17,25 @@
      All sessions NEED to have a closing activity, reflection, discussion
 ## Session Outline 
 
-(Please include who is facilitating and how much time you are going to spend in each section):
+  1. intro to topic by John
```

Comment:
Why are you starting with scooters?  Be clear on what you are hoping to get out of this debate as this might not be a "problem" people have thought about before.